xmpp_http_upload.go

```diff
@@ -0,0 +1,43 @@
+package xmpp
+
+import (
+	"encoding/xml"
+)
+
+const (
+	XMPPNS_HTTP_UPLOAD = "urn:xmpp:http:upload:0"
```

Comment:
Can you move these constants to the other constants in xmpp.go? I am still planning to move all the constants into a const.go file to make them more discoverable.